ut/client_ut.cpp

```diff
@@ -1196,6 +1197,22 @@ INSTANTIATE_TEST_SUITE_P(ClientLocalReadonly, ReadonlyClientTest,
     }
 ));
 
+INSTANTIATE_TEST_SUITE_P(ColumnNames, AbnormalColumnNamesTest,
```

Comment:
IMO it makes sense to move this test suite instantiation right to the definition of the test itself (`ut/abnormal_column_names_test.cpp`). That would make things a bit simpler.